terraform/README.md

```diff
@@ -1,31 +1,35 @@
 # Payment Reconciler - terraform 
 
 ## Introduction
-This provisions the payment-reconciler Lambda function which is triggered by the cloud watch event.
+* This provisions the `payment-reconciler` Lambda function, together with a cloud watch event to trigger it.
+* Use [the Concourse pipeline](https://ci.platform.aws.chdev.org/teams/development/pipelines/payment-reconciler) to
```

Comment:
I would remove all mention of the pipeline as it's not relevant. This README should focus on what the project itself

Reply:
This is a README for the terraform/deployment aspects of the project only. I had thought that we were trying to emphasise that people should be carrying out automated deployments from Concourse. Hence its mention in this context.

Reply:
My concern is that the URLs can change and that would leave a broken link on the README.

Reply:
I will remove the URL from here then and move it to the Confluence page. 👍